frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -190,11 +192,9 @@ class TableInteractive extends Component {
       column => column.source === "aggregation",
     );
     const isNotebookPreview = this.props.queryBuilderMode === "notebook";
-    const newShowDetailState = !(
-      isPivoted ||
-      hasAggregation ||
-      isNotebookPreview
-    );
+    const newShowDetailState =
```

Comment:
I understand this flag is for the object detail icon here, why do we want to hide it on the SDK?
![image](https://github.com/metabase/metabase/assets/1937582/a2c9e72d-fa17-41e9-a752-30df139de1eb)